themes/default/content/docs/get-started/install/_index.md

```diff
@@ -153,12 +313,33 @@ To update Pulumi to a more recent version:
 > winget upgrade pulumi
 ```
 
-### Standalone Installer
-
-You can download the latest [Pulumi Installer for Windows x64](https://github.com/pulumi/pulumi-winget/releases) and run it like any other installer. It will automatically add Pulumi to the path and make it available machine-wide.
-
-### Installation Script
-
+</div>
+</div>
+
+<div class="accordion-item text-2xl py-3 border-t-2">
+<input type="checkbox" class="absolute hidden" id="windows-standalone-installer" />
+<label for="windows-standalone-installer" class="accordion-label">
+<h5 class="mt-2 w-2/3">Standalone Installer (MSI)</h5>
+<div class="flex flex-grow justify-end items-center">
+<span class="closed-accordion">+</span>
+<span class="open-accordion hidden">-</span>
+</div>
+</label>
+<div class="accordion-item-body-no-animation text-base">
+You can download the latest [Pulumi Installer for Windows x64](https://github.com/pulumi/pulumi-winget/releases/download/{{< latest-version >}}/pulumi-{{< latest-version >}}-windows-x64.msi) and run it like any other installer. It will automatically add Pulumi to the path and make it available machine-wide.
```

Comment:
```suggestion
Download the latest [Pulumi Installer for Windows x64](https://github.com/pulumi/pulumi-winget/releases/download/{{< latest-version >}}/pulumi-{{< latest-version >}}-windows-x64.msi) and run it like any other installer. It will automatically add Pulumi to the path and make it available machine-wide.
```